src/Core/Prefab/GlobeView.js

```diff
@@ -163,6 +164,31 @@ export function createGlobeLayer(id, options) {
         return false;
     }
 
+    // return common extent to pool the geometries
+    // the geometry in common extent is identical to the existing input
+    // with a transformation (translation, rotation)
+    wgs84TileLayer.getCommonGeometryExtent = (extent) => {
+        const sizeLatitude = Math.abs(extent.west() - extent.east()) / 2;
+        const communExtent = new Extent(extent.crs(), -sizeLatitude, sizeLatitude, extent.south(), extent.north());
+        communExtent._internalStorageUnit = extent._internalStorageUnit;
+        return communExtent;
+    };
+
+    const axisZ = new THREE.Vector3(0, 0, 1);
+    const axisY = new THREE.Vector3(0, 1, 0);
+    const quatToAlignLongitude = new THREE.Quaternion();
+    const quatToAlignLatitude = new THREE.Quaternion();
+
+    // return rotation to transform tile to position it on ellipsoid
+    // this transformation take into account the transformation of the parents
+    wgs84TileLayer.getQuaternionTile = (tile) => {
```

Comment:
Wrong naming: the name doesn't express what the function does.
And the argument should be an extent, since you don't use any other properties of the `tile`.

Reply:
changed in getQuaternionFromExtent